docs/api/puppeteer.baselaunchoptions.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_label: BaseLaunchOptions
+---
+
+# BaseLaunchOptions interface
+
+Generic launch options that can be passed when launching any browser.
+
+#### Signature:
+
+```typescript
+export interface BaseLaunchOptions
+```
+
+## Properties
+
+| Property           | Modifiers             | Type                                                        | Description                                                                                                                                                                                                                           | Default                                   |
+| ------------------ | --------------------- | ----------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------- |
+| channel            | <code>optional</code> | [ChromeReleaseChannel](./puppeteer.chromereleasechannel.md) | Chrome Release Channel                                                                                                                                                                                                                |                                           |
```

Comment:
Is `channel` used for FF as well?

Reply:
Maybe explicitly specify which options work in Chrome, FF or both?

Reply:
I think it's an existing feature/bug